components/brave_rewards/browser/database/publisher_info_database.cc

```diff
@@ -359,6 +365,47 @@ bool PublisherInfoDatabase::RestorePublishers() {
   return restore_q.Run();
 }
 
+bool PublisherInfoDatabase::GetExcludedList(
+    ledger::PublisherInfoList* list) {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  CHECK(list);
```

Comment:
CHECK will throw an exception on release builds. Recommend to change to:```DCHECK(list);
if (!list) {
  return false;
}```

Reply:
not part of the PR, but fixed